<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiskManagerClass</class>
 <widget class="QMainWindow" name="DiskManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiskManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>これより古いファイルを移動・削除</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxYear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2000</number>
     </property>
     <property name="maximum">
      <number>2099</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>年</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>月</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMonth">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>日</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxDay">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>時</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxHour">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>分</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxMinute">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxSecond">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxExecuteDelete">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>251</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>削除を実行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSaveSetting">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>設定保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTransmitSetting">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>設定送信</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAutoMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>自動転送・削除</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>811</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>手動での操作（リストをダブルクリック）</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetFileList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>791</width>
       <height>311</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>パス</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ファイル名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>サイズ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>更新日時</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidgetSourceDrive">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>411</width>
      <height>321</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>PC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Drive</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Free area</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>監視対象Drive</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
